src/utils/save.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+
+
+def save_dataframe(
+        data: pd.DataFrame,
+        filename: str,
+        destination: str = "./data"):
```

Comment:
All dataframes will be saved inside ./data, so maybe we should only allow to specify subdir.